src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetProjectManagerService.cs

```diff
@@ -175,11 +176,11 @@ public async ValueTask<IProjectContextInfo> GetProjectAsync(string projectId, Ca
             // If this is a PR-style project, get installed and transitive package references. Otherwise, just get installed package references.
             var prStyleTasks = new List<Task<ProjectPackages>>();
             var nonPrStyleTasks = new List<Task<IEnumerable<PackageReference>>>();
-            foreach (NuGetProject? project in projects)
+            foreach (NuGetProject? project in projects) // can this be parallel ?
```

Comment:
Is this a TODO?

Reply:
No, just an idea to possibly improve performance. Not going to cover it in this PR.